result/isislsp.yaml

```diff
@@ -0,0 +1,380 @@
+components:
+  schemas:
+    IsisLsps.State.Request:
+      description: >- 
+        The request to retrieve ISIS Link State PDU (LSP) information learned by the router.
+      type: object
+      properties:
+        isis_router_names:
+          description: >-
+            The names of ISIS routers for which learned information is requested.
+            An empty list will return results for all ISIS routers.
+          type: array
+          items:
+            type: string
+          x-constraint:
+            - "/components/schemas/Device.IsisRouter/properties/name"
+  
+    IsisLsps.State:
+      description: >-
+        The result of ISIS LSP information that are retrieved.
+      type: object
+      properties:
+        isis_router_name:
+          description: >-
+            The name of the ISIS Router.
+          type: string
+        lsps:
+          description: >-
+            One or more LSPs that are learned by this ISIS router.
+          type: array
+          items:
+            $ref: '#/components/schemas/IsisLsp.State'
+
+    IsisLsp.State:
+      description: >-
+        ISIS LSP.
+      type: object
+      properties:
+        lsp_id:
+          description: >-
+            LSP ID in the format, e.g. '640000000001-00-00'.
+            LSP ID consists of the System ID of a neighbor, the Pseudonode ID, and the LSP number.
+            The last two bytes represent Pseudonode ID and LSP number respectively.
+            A pseudonode is a logical representation of the LAN which is generated by a Designated Intermediate System (DIS)
+            on a LAN segment. If one LSP exceeds the maximum LSP size then it is sent in another LSP with the LSP number incremented by one.
+            A router's learned LSP gets refreshed by 'remaining_lifetime'. Then the sequence number is incremented by 1.
+          type: string
+        pdu_type:
+          description: >-
+            Link State PDU type.
+          type: string
+          enum: [level_1, level_2]
+        remaining_lifetime:
+          description: >-
```

Comment:
properties marked "mandatory" in yang should be indicated with "required" attribute to indicate MUST in the otg-models as well.